lib/v2/screens/sign_up/claim_invite/mappers/claim_invite_mapper.dart

```diff
@@ -12,26 +12,34 @@ class ClaimInviteMapper extends StateMapper {
 
     if (result.isError) {
       return currentState.copyWith(
-          claimInviteState: ClaimInviteState.error(claimInviteCurrentState, 'No invites found, try another code'.i18n));
+        claimInviteState: claimInviteCurrentState.copyWith(
+          claimInviteView: ClaimInviteView.fail, // gray screen
+          pageCommand: ShowErrorMessage(''), // dialog
+          errorMessage: 'No invites found, try another code'.i18n,
+        ),
+      );
     } else {
       final InviteModel inviteModel = result.asValue!.value;
 
       if (inviteModel.account.isNullOrEmpty) {
         // invite code is valid and can be claimed
-        final newState = claimInviteCurrentState.copyWith(
```

Comment:
According to Fazard's logic if this (`inviteModel.account`) arrives empty it means that it is succes. 
Could you @n13 confirm if this is correct?

Reply:
Yes! Correct.

Reply:
Claiming an invite sets the account - so one that has been claimed already has account set (also has secret set to something other than 0000....)